shared/js/participantStore.js

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var loop = loop || {};
+loop.store = loop.store || {};
+
+loop.store.ParticipantStore = (function() {
+  "use strict";
+
+  const PARTICIPANT_SCHEMA = {
+    name: "",
```

Comment:
Does having `name` work correctly? I think `Actions` actually have a `name` of the action itself, e.g., `updatedParticipant` and `updatedPresence`.